### PR TITLE
update style in user card and responsive view

### PR DESCRIPTION
**Pull request by raaj-oo7**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -163,59 +147,42 @@ ul {
...
-        display: grid;
-        grid-template-columns: 1fr;
+    .user-image {
+        height: 80px;
+        width: 80px;
     }
 
     .container {
-        padding: 10px;
+        padding: 6% 5%;
```
Comment (SujalXplores) [inline style.css]: @raaj-oo7 don't use `%` for padding, always use `px`

**Review by SujalXplores — CHANGES_REQUESTED** [review]

**Comment by SujalXplores** [comment]
@raaj-oo7 Look for the details the colors of the Reveal button is different for initial card and after reveal click
It should be consistent
image

File: index.html [hunk 2]
```diff
@@ -4,29 +4,55 @@
...
+    <link rel="icon" type="image/x-icon" href="./image/zuru-logo.svg">
     <link rel="stylesheet" href="./style.css">
     <title>ZURU</title>
 </head>
 
 <body>
-    <h3 class="heading">ZURU</h3>
+    <div class="app-content">
+        <h3 class="heading">ZURU</h3>
+        <!-- user container -->
```
Comment (SujalXplores) [inline index.html]: @raaj-oo7 This comment is not needed as the class name already pointing to the reference container itself
Give comments only when needed for example complex logic or block of code that is difficult to understand at first glance

File: main.js [hunk 3]
```diff
@@ -16,42 +17,70 @@ async function fetchFollowers(username) {
   }
```
Comment (SujalXplores) [inline main.js:17]: @raaj-oo7 This fetch function catch block should not throw an error.
Could you please explain why this function throws error?

File: style.css [hunk 4]
```diff
@@ -9,26 +9,33 @@ body {
...
 }
 
+::-webkit-scrollbar {
+    width: 0;
+    background: transparent;
+}
+
 .heading {
     font-weight: bold;
     font-size: x-large;
```
Comment (SujalXplores) [inline style.css]: @raaj-oo7 always declare font size in px

**Review by SujalXplores — CHANGES_REQUESTED** [review]

File: main.js (hunk 3 above)
Comment (raaj-oo7) [reply main.js:17]: if network error or fetch operation error occurred that time function throws an error

**Pull request merged**